release/tune_tests/cloud_tests/tune_cloud_tests.yaml

```diff
@@ -28,6 +28,26 @@
     prepare: python wait_cluster.py 4 600
     script: python workloads/run_cloud_test.py durable_upload --bucket s3://data-test-ilr/durable_upload
 
+- name: aws_durable_upload_rllib_str
+  cluster:
+    app_config: app_config_ml.yaml
+    compute_template: tpl_aws_4x2.yaml
+
+  run:
+    timeout: 600
+    prepare: python wait_cluster.py 4 600
+    script: python workloads/run_cloud_test.py durable_upload --trainable rllib_str --bucket s3://data-test-ilr/durable_upload_rllib_str
+
+- name: aws_durable_upload_rllib_trainer
+  cluster:
+    app_config: app_config_ml.yaml
+    compute_template: tpl_aws_4x2.yaml
+
+  run:
+    timeout: 600
+    prepare: python wait_cluster.py 4 600
+    script: python workloads/run_cloud_test.py durable_upload --trainable rllib_trainer --bucket s3://data-test-ilr/durable_upload_rllib_trainer
+
```

Comment:
This adds the release test

Reply:
Yes - please note that it is currently commented out in the `build_pipeline.py` so not included in nightly release testing, yet. This will be dealt with in a follow-up PR that deals with the bookkeeping issues